### PR TITLE
Configurable icon set

### PR DESCRIPTION
**Pull request by jaahon**
#### Summary

Make used icon set easily configurable and build the frontend bundles with pro icons in CI.

#### Testing instructions

Read the instructions in the frontend README and try using the pro icon set.

### CODE REVIEW

File: frontend/packages/icons/setup.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+fi
+
+if [ "$ICON_SET" = 'pro' ]; then
+  echo "Setting up pro version of Font Awesome icons"
+  cat "$ICONS_DIR/pro.icons.ts" > "$ICONS_DIR/icons.ts"
+  cat "$ICONS_DIR/pro.package.json" > "$ICONS_DIR/package.json"
+else
+  echo "Setting up free version of Font Awesome icons"
+  cat "$ICONS_DIR/free.icons.ts" > "$ICONS_DIR/icons.ts"
+  cat "$ICONS_DIR/free.package.json" > "$ICONS_DIR/package.json"
```
Comment (mikkopiu) [inline frontend/packages/icons/setup.sh:21]: Why not `cp`?
```suggestion
  cp -a "$ICONS_DIR/pro.icons.ts" "$ICONS_DIR/icons.ts"
  cp -a "$ICONS_DIR/pro.package.json" "$ICONS_DIR/package.json"
else
  echo "Setting up free version of Font Awesome icons"
  cp -a "$ICONS_DIR/free.icons.ts" "$ICONS_DIR/icons.ts"
  cp -a "$ICONS_DIR/free.package.json" "$ICONS_DIR/package.json"
```

File: frontend/packages/icons/free.package.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@evaka/icons",
+  "description": "Icons used in evaka",
+  "version": "1.0.0",
+  "private": true
```
Comment (mikkopiu) [inline frontend/packages/icons/free.package.json:5]: Missing `license` field

File: frontend/packages/icons/pro.package.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@evaka/icons",
+  "description": "Icons used in evaka",
+  "version": "1.0.0",
+  "private": true,
```
Comment (mikkopiu) [inline frontend/packages/icons/pro.package.json:5]: Missing `license` field

File: frontend/packages/icons/setup.sh (hunk 1 above)
Comment (jaahon) [reply frontend/packages/icons/setup.sh:21]: No reason really, it was the first thing that popped into my mind.

**Pull request closed without merging**